content/en/blog/release-v2.3.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Unveiling Kairos v2.3 - FIPS Compliance and More!"
+date: 2023-07-10
+linkTitle: "Unveiling Kairos v2.3"
+description: "Unveiling Kairos v2.3 - FIPS Compliance and More!"
+author: Mauro Morales ([Twitter](https://twitter.com/mauromrls)) ([GitHub](https://github.com/mauromorales))
+---
+<h1 align="center">
+  <br>
+     <img width="184" alt="kairos-white-column 5bc2fe34" src="https://user-images.githubusercontent.com/2420543/215073247-96988fd1-7fcf-4877-a28d-7c5802db43ab.png">
+    <br>
+<br>
+</h1>
+
+Excitement abounds as we roll out Kairos v2.3, the latest iteration of our project, packed with new features and enhancements. This update shines a spotlight on the Federal Information Processing Standards (FIPS).
+
+FIPS, a set of standards governing document processing, encryption algorithms, and other IT standards for non-military government agencies and contractors, is now an integral part of Kairos. The addition of a FIPS generic and Ubuntu-specific framework image marks a significant milestone in our journey to offer a secure and reliable Linux distribution. Furthermore, we've furnished examples to facilitate building from scratch on Ubuntu, Fedora, and RockyLinux.
+
+This release also heralds a series of updates to the diverse Kairos flavors. The Ubuntu core images now stand at version 23.04, AlmaLinux has joined our flavor offerings, and openSUSE Leap has been upgraded to version 15.5.
+
+In our relentless pursuit of perfection, several bugs have been squashed to ensure a seamless experience for our users. The issue of /tmp being remounted under alpine has been addressed, the problem with restoring machine-id on reboot has been rectified, and qemu-guest-agent has been added to images where it was previously absent.
```

Comment:
```suggestion
Several bugs have been squashed to ensure a seamless experience for our users. The issue of /tmp being remounted under alpine has been addressed, the problem with restoring machine-id on reboot has been rectified, and qemu-guest-agent has been added to images where it was previously absent.
```

I like the chatgpt tone, but it seems it pushed too much here :D

Reply:
yup was part of the experiment but didn't want to mention just to know how much was good or bad :) ... ok will re-write thanks!